### ISSUE TITLE
Problem with RTL Tables

### ISSUE THREAD

**Issue by alex2230** [mkoryak/floatThead#449]
Hi,
I know that: "RTL is not really supported by floatThead"
floatThead works well on Chrome, and has no problems with RTL tables, But floatThead+RTL tables are not working properly on Firefox.

Is there a way to fix the problem with Firefox?

Please see the links below with Firefox:
LTR: https://codepen.io/alex200/pen/XWWqLaQ
RTL: https://codepen.io/alex200/pen/VwwxJWe

**Comment by mkoryak** [comment]
sorry, this fell off my radar. Ill look soon

**Comment by mkoryak** [comment]
I did some digging. The culprit is this offset call:
https://github.com/mkoryak/floatThead/blob/master/src/jquery.floatThead.js#L711

it returns different values in RTL on firefox vs chrome. Digging further it looks like 
`getBoundingClientRect` on table returns different values on those browsers:

```
console.log('wat', $('#table_demo')[0].getBoundingClientRect())
```

firefox returns a negative value while chrome returns 0 or positive.

Do you still care about having this fix :) ?

**Comment by alex2230** [comment]
@mkoryak 
Thanks for reply.

The jquery.rtl-scroll-type plugin adjusts the RTL-table header correctly, but when user scrolls left and right, the header does not change and is fixed!

Please see the links below with Firefox And screen width smaller than 900 pixels:
https://codepen.io/alex200/pen/MWWNMzQ

**Comment by mkoryak** [comment]
That's fun, I should have guessed some other calculations would need to be inverted.
I'm upgrading this to a bug.

**Comment by swdreams** [comment]
Hi,
I like this repository, but it's pity that it doesn't support a functionality to freeze left column(s).
Is it possible or are there any good source?
Thanks

**Comment by mkoryak** [comment]
No, this plugin does not do this and there are no plans for it.

Feel free to write your own :)
